src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -1083,7 +1082,7 @@ private void Canonicalize()
             bool done;
 
             _canonical = true;
-            _rangelist.Sort(SingleRangeComparer.Instance);
+            _rangelist.Sort(SingleRangeComparer.s_instance);
```

Comment:
Style wise I don't know how we feel about directly accessing fields, even of nested classes.

Reply:
Oh I see we were already doing it, you renamed. Fine.